src/libraries/System.Runtime/tests/System/TimeSpanTests.cs

```diff
@@ -142,6 +142,16 @@ public static void Add_Invalid()
             Assert.Throws<OverflowException>(() => TimeSpan.MinValue + new TimeSpan(-1)); // Result < TimeSpan.MinValue
         }
 
+        [Fact]
+        public static void FromMinMaxValue_DoesNotThrow()
+        {
+            var maxTimeSpan = TimeSpan.FromDays(TimeSpan.MaxValue.TotalDays);
+            var minTimeSpan = TimeSpan.FromDays(TimeSpan.MinValue.TotalDays);
```

Comment:
Our usual stance for what makes something "obvious", and thus permits `var`, is only new/cast/as-cast. (No factories, no "but everybody knows").

While I'll grant it's obvious in this context, let's go ahead and specify the type name.

Reply:
Alright, see 4693fe339cfd68ddab97674d42b8d5d4fc4d37d0.